src/Button/__tests__/BasicButton.test.tsx

```diff
@@ -1,29 +1,37 @@
 import React from 'react';
-import { render, fireEvent, cleanup } from 'test-utils';
-import { spy } from 'sinon';
+import { render, fireEvent } from 'test-utils';
+import { spy, SinonSpy } from 'sinon';
 import assert from 'assert';
 import BasicButton from '../BasicButton';
 
 describe('Basic Button', function () {
   let getByText;
```

Comment:
`getByText` should also have a type declaration in all of these tests.

Reply:
@jonseitz I haven't been able to find a type for getByText. I tried to use the type ```GetByText```, but it gave me the error ``` Types of parameters 'text' and 'container' are incompatible``` so it looks like this is not the right type to use. 
I ended up declaring a type using the function declaration listed here: [https://testing-library.com/docs/dom-testing-library/api-queries#bytext](url)
Is the way that I did this through commit ddf4baf acceptable?

Reply:
I ended up using `BoundFunction<GetByText>;` as the type for `getByText`